e2e/packages/test-data/generate-test-data.ts

```diff
@@ -86,10 +87,7 @@ const logs = (
 ).flatMap((range) => range.logs);
 
 console.log("writing logs to", logsFilename);
-await fs.writeFile(
-  logsFilename,
-  JSON.stringify(logs, (k, v) => (typeof v === "bigint" ? v.toString() : v), 2)
-);
```

Comment:
manually transforming `bigint`s didn't work well because we want to pass this into functions that expect `Log` (which expect `bigint` not strings)

Reply:
ended up moving to rpc logs instead, which are json serializable